segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AdminRequestProcessorImpl.java

```diff
@@ -120,5 +124,33 @@ public void flushToStorage(WireCommands.FlushToStorage flushToStorage) {
                 .exceptionally(ex -> handleException(flushToStorage.getRequestId(), null, operation, ex));
     }
 
+    @Override
+    public void listStorageChunks(WireCommands.ListStorageChunks listStorageChunks) {
+        final String operation = "ListStorageChunks";
+        final String segment = listStorageChunks.getSegment();
+
+        if (!verifyToken(segment, listStorageChunks.getRequestId(), listStorageChunks.getDelegationToken(), operation)) {
+            return;
```

Comment:
You can check the `AdminRequestProcessorAuthFailedTest` to see how this line exactly works. In case the token is invalid then the method call to `verifyToken` fails and throws the required exception.